ui/src/components/yaml/styles.module.less

```diff
@@ -208,9 +208,7 @@
   .yaml_content_diagnosisPanel {
     box-sizing: border-box;
```

Comment:
The min-height property is removed, which may cause layout issues when content is minimal.